### DISCUSSION TITLE
Merged ground markers.

### DISCUSSION THREAD

**Discussion by geheur** [runelite/runelite#17158] [Ideas]
Looks good imo.

![image](https://github.com/runelite/runelite/assets/41499327/0f2a517d-4462-4543-a6f0-13d1c3815627)

https://github.com/geheur/runelite/commit/4333865e734108c2bd62330fe09f343214c77465 (just proof of concept, has some bad code 
 such as the O(n^2) section, although it doesn't seem to outgrow the time it takes to render them even at 500 ground markers.)

**Comment by InfernoStats** [comment]
I think this suggestion would look better without any of the line artifacts in the joined sections.

**Reply by geheur** [reply]
That can be optional.
![image](https://github.com/runelite/runelite/assets/41499327/0711dea2-9a07-428e-95fe-f4a420ea8f81)

**Reply by geheur** [reply]
Also it's not an artifact, it's intended so you can see where the tiles are.

**Reply by InfernoStats** [reply]
Perhaps artifacts wasn't the right word, but it did look odd and I'd imagine if you want to join them you're looking for more of a poly area effect than individual tiles. I worked out something similar a long time ago (faked via 2 polygons)  and have been using it for a cleaner Xarpus room
![image](https://github.com/runelite/runelite/assets/77599829/f1fb4873-bba2-433a-b01c-ad0243ef3581)

**Reply by geheur** [reply]
Looks good. you can do something similar with the line markers plugin but it seems to be a pain in the ass to set up.
